index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="style.css">
+	<meta charset="UTF-8">
+	<title>Microverse trials</title>
+</head>
+<body>
+<main id="main">
+	<div id="title-container">
+		<h1 id="title"> Lionel Andres Messi Cuccittini </h1> 
+		<h3> The new king after Maradona </h3>
```

Comment:
your code: <h3*> The new king after Maradona </h3*>

I think you should put <h2*> The new king after Maradona </h2*> instead. Think h2 as a subsection and h3 as a sub-subsection. Also it would make bigger the letters and more easy to understand for you when changes has to be made.

Reply:
- [ ] I can see here the student used `h1` tag for the most important title on the page, it looks like a best practice in HTLM 👌🏼👍🏼, please consider adding relevant comments about a mistake or an enhancement that you think the student needs to follow 💯 .